src/KeyVault/KeyVault.Sdk/README.md

```diff
@@ -0,0 +1,39 @@
+# Overall
+This directory contains management plane service clients of Az.KeyVault module.
+
+## Run Generation
+In this directory, run AutoRest:
+```
+autorest --reset
+autorest --use:@microsoft.azure/autorest.csharp@2.3.90
+autorest.cmd README.md --version=v2
+```
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+``` yaml
+csharp: true
+clear-output-folder: true
+openapi-type: arm
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+```
+
+###
+``` yaml
+input-file:
+  - https://github.com/Azure/azure-rest-api-specs/blob/fead0dec636e7554fb8401370418085136d4f052/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-07-01/common.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/fead0dec636e7554fb8401370418085136d4f052/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-07-01/keys.json
```

Comment:
please try to remove this one because it is not used by Azure PowerShell

Reply:
Hi @dingmeng-xue , updated based on comments. Please check it.